src/sentry/relay/config/__init__.py

```diff
@@ -65,6 +65,7 @@
     "projects:span-metrics-extraction",
     "projects:span-metrics-extraction-addons",
     "organizations:indexed-spans-extraction",
+    "organizations:ingest-spans-in-eap",
```

Comment:
Does this flag need to be exposed to relay?

Reply:
Yes, I'm making a PR to Relay too so we can add a field to the span payload to let the Snuba EAP consumers know they should ingest the span or not.